postgresql/config/functions.sh

```diff
@@ -67,10 +67,18 @@ bootstrap_replica_via_pg_basebackup() {
 }
 
 ensure_dir_ownership() {
-  echo 'Making sure hab user owns var and data paths'
-  chown -RL hab:hab {{pkg.svc_var_path}}
-  chown -RL hab:hab {{pkg.svc_data_path}}
-  chmod 0700 {{pkg.svc_data_path}}/pgdata
+  if [[ $EUID -eq 0 ]]; then
+    # if EUID is root, so we should chown to pkg_svc_user:pkg_svc_group
+    the_user={{pkg.svc_user}}
```

Comment:
readability nit:  the `the` in `the_user/group` is ambiguous.  suggestion:  `effective_user` or `service_user`?